sdk/identity/Azure.Identity/src/AuthorizationCodeCredential.cs

```diff
@@ -85,8 +85,8 @@ internal AuthorizationCodeCredential(string tenantId, string clientId, string cl
             Argument.AssertNotNull(authorizationCode, nameof(authorizationCode));
             _clientId = clientId;
             _authCode = authorizationCode ;
-            _options = options ??= new TokenCredentialOptions();
-            _pipeline = CredentialPipeline.GetInstance(options);
+            _allowMultiTenantAuthentication = options?.AllowMultiTenantAuthentication ?? true;
```

Comment:
Shouldn't the default of _allowMultiTenantAuthentication be false if no options are specified?